cli/cmd/cluster_prepare.go

```diff
@@ -160,170 +220,72 @@ func (r *runners) prepareCluster(_ *cobra.Command, args []string) error {
 		return errors.Wrap(err, "get app")
 	}
 
-	// create a test customer with the correct entitlement values
-	email := fmt.Sprintf("%s@relicated.com", clusterName)
-	customerOpts := kotsclient.CreateCustomerOpts{
-		Name:        clusterName,
-		ChannelID:   "",
-		AppID:       a.ID,
-		LicenseType: "test",
-		Email:       email,
-	}
-	customer, err := r.api.CreateCustomer(r.appType, customerOpts)
-	if err != nil {
-		return errors.Wrap(err, "create customer")
-	}
-
-	_, err = fmt.Fprintf(r.w, "Customer %s (%s) created.\n", customer.Name, customer.ID)
-	if err != nil {
-		return errors.Wrap(err, "write to stdout")
+	var entitlements []kotsclient.EntitlementValue
+	for _, set := range r.args.prepareClusterEntitlements {
+		setParts := strings.SplitN(set, "=", 2)
+		if len(setParts) != 2 {
+			return errors.Errorf("invalid entitlement %q", set)
+		}
+		entitlements = append(entitlements, kotsclient.EntitlementValue{
+			Name:  setParts[0],
+			Value: setParts[1],
+		})
 	}
 
 	// wait for the wait group
 	wg.Wait()
 
 	log.ActionWithSpinner("Waiting for release to be ready")
-	isReleaseReady, err := isReleaseReadyToInstall(r, log, *release)
-	if err != nil || !isReleaseReady {
+	appRelease, err := getReadyAppRelease(r, log, *release)
+	if err != nil || appRelease == nil {
 		log.FinishSpinnerWithError()
 		return errors.Wrap(err, "release not ready")
 	}
```

Comment:
this will return nil if err == nil.

```suggestion
	if err != nil || appRelease == nil {
		log.FinishSpinnerWithError()
		if err == nil {
			return errors.New("release not ready")
		}
		return errors.Wrap(err, "release not ready")
	}
```